src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -60,33 +60,31 @@ const TransferDescription = ({ value = '', symbol, recipient }: TransferDescProp
       {' '}
       to:
     </Bold>
-    <br />
     <EtherscanLink type="address" value={recipient} />
-  </Paragraph>
+  </Block>
 )
 
 const SettingsDescription = ({ removedOwner, addedOwner, newThreshold }: DescriptionDescProps) => (
   <>
     {removedOwner && (
-      <Paragraph data-testid={TRANSACTIONS_DESC_REMOVE_OWNER_TEST_ID}>
+      <Block data-testid={TRANSACTIONS_DESC_REMOVE_OWNER_TEST_ID}>
```

Comment:
Why block? This is text

Reply:
It has the component `EtherscanLink`, which has the component `Block` as his root element

Reply:
`EtherscanLink` uses an anchor tag, not Block

Reply:
https://github.com/gnosis/safe-react/blob/137-txlist-improvements/src/components/EtherscanLink/index.jsx

Reply:
👌